porcupine/default_filetypes.toml

```diff
@@ -359,6 +359,7 @@ syntax_highlighter = "tree_sitter"
 tree_sitter_language_name = "json"
 indent_size = 2
 tabs2spaces = true
+autoindent_regexes = {dedent = '.*[^,{[]\n', indent = '.*[{[]\n'
```

Comment:
```suggestion
autoindent_regexes = {dedent = '.*[^,{[]\n', indent = '.*[{[]\n'}
```

Reply:
I was doing this in the builtin GH editor so an issue was kinda to be expected.